autotx/cli.py

```diff
@@ -57,18 +54,13 @@ def run(prompt: str, non_interactive: bool):
         send_eth(dev_account, manager.address, 10, web3)
         print(f"Sent 10 ETH to smart account for testing purposes")
 
-    print("Starting smart account balances:")
-    show_address_balances(web3, network_info, manager.address)
```

Comment:
We don't want this anymore?

Reply:
this is something i forgot to mention you - this function shows the balance of every token in the `network.tokens` list... So it now takes some time (a lot tbh) to show every token balance; I am not sure what the best approach would be

Reply:
after talking with jure we've decided to show only the balance of the default tokens && when there's not smart account set